client/src/components/BlogCard.jsx

```
@@ -0,0 +1,26 @@
+      onClick={() => navigate(`/blog/${_id}`)}
+      className='w-full rounded-lg overflow-hidden shadow hover:scale-102 hover:shadow-primary/25 duration-300 cursor-pointer'
+    >
+      <img src={image} alt='test' className='aspect-video' />
+      <span className='ml-5 mt-4 px-3 py-1 inline-block bg-primary/20 rounded-full text-primary text-xs'>
+        {category}
+      </span>
+      <div className='p-5'>
+        <h5 className='mb-2 font-medium text-gray-900'>{title}</h5>
+        <p
+          className='mb-3 text-xs text-gray-600'
+          dangerouslySetInnerHTML={{ __html: description.slice(0, 80) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)